### DISCUSSION TITLE
How to disabled :hover :fouce etc if need to do;

### DISCUSSION THREAD

**Discussion by marol1210** [tailwindlabs/tailwindcss#10341] [Help] [answered]
**Problem:**  wechat's  miniprogram wxss file don't support  special character , '*'  '.py-2\.5' etc.

![image](https://user-images.githubusercontent.com/7519301/212838210-ac45012e-256d-4aa2-a2ef-50c1e961fc29.png)

Action:   Disabling Preflight , in tailwindcss.config.js

> module.exports = {
>   ....
>  content: {
>	  files: ["./pages/**/*.vue"]
>  },
>   corePlugins: {
>     preflight: false,
>   }
>   ....
> }
    **Result:**  
![image](https://user-images.githubusercontent.com/7519301/212839050-65416b5a-65e0-4c39-b22d-563d6046abd3.png)

**Question:**  why there had style content if already disabled Preflight?

**Comment by wongjn** [comment, marked as answer]
Those are the CSS custom property default values which are needed for some of the utility classes to work properly. You could try setting `experimental.optimizeUniversalDefaults` to `true` to optimize this rule, and I believe this would remove the rule with the asterisk.

**Reply by marol1210** [reply]
you are right , great!    thanks.
